app/models/policy/auction.rb

```diff
@@ -0,0 +1,263 @@
+class Policy::Auction
+  BID_LIMIT = 3500
+  BID_INCREMENT = 1
+
+  include ActionView::Helpers::NumberHelper
```

Comment:
If you can avoid includes like this, it would be good. When you include something this big into the object, it is no longer a PORO. I usually try to find a way to either directly call the module method, but Rails sometimes makes this very difficult. Or, I put all the inclusions in a separate class who can be used to isolate POROs from Rails dependencies. 

The place you can first feel the non-PORO pain is in tests. This object can't be tested with just rspec alone. It needs ActionView, and likely a lot of Rails to run. 

With a direct call to the module, you can mock out that object in pure specs. With an dedicated object that deals with those inclusions, you can mock out the whole object in tests.